apps/web/playwright/managed-event-types.e2e.ts

```diff
@@ -1,212 +1,246 @@
-import type { Page } from "@playwright/test";
+import type { Locator, Page } from "@playwright/test";
 import { expect } from "@playwright/test";
+import { apiLogin } from "playwright/fixtures/users";
 
-import { IS_TEAM_BILLING_ENABLED } from "@calcom/lib/constants";
+import { SchedulingType } from "@calcom/prisma/enums";
 
-import { test } from "./lib/fixtures";
-import {
-  bookTimeSlot,
-  fillStripeTestCheckout,
-  localize,
-  selectFirstAvailableTimeSlotNextMonth,
-} from "./lib/testUtils";
+import { test, type Fixtures } from "./lib/fixtures";
+import { bookTimeSlot, localize, selectFirstAvailableTimeSlotNextMonth } from "./lib/testUtils";
 
-test.afterEach(({ users }) => users.deleteAll());
+test.afterEach(async ({ users }) => {
+  await users.deleteAll();
+});
+
+/** So we can test different areas in parallel and avoiding the creation flow each time */
+async function setupManagedEvent({
+  users,
+  unlockedFields,
+}: {
+  users: Fixtures["users"];
+  unlockedFields?: Record<string, boolean>;
+}) {
+  const teamMateName = "teammate-1";
+  const teamEventTitle = "Managed";
+  const adminUser = await users.create(null, {
+    hasTeam: true,
+    teammates: [{ name: teamMateName }],
+    teamEventTitle,
+    teamEventSlug: "managed",
+    schedulingType: "MANAGED",
+    addManagedEventToTeamMates: true,
+    managedEventUnlockedFields: unlockedFields,
+  });
+  // eslint-disable-next-line @typescript-eslint/no-non-null-assertion
+  const memberUser = users.get().find((u) => u.name === teamMateName)!;
+  const { team } = await adminUser.getFirstTeamMembership();
+  const managedEvent = await adminUser.getFirstTeamEvent(team.id, SchedulingType.MANAGED);
+  return { adminUser, memberUser, managedEvent, teamMateName, teamEventTitle };
+}
+
+const getByKey = async (page: Page, key: string) => page.getByText((await localize("en"))(key));
 
 test.describe("Managed Event Types", () => {
   test("Can create managed event type", async ({ page, users }) => {
     // Creating the owner user of the team
-    const adminUser = await users.create();
+    const adminUser = await users.create(null, {
+      hasTeam: true,
+      teammates: [{ name: "teammate-1" }],
+    });
     // Creating the member user of the team
-    const memberUser = await users.create();
     // First we work with owner user, logging in
     await adminUser.apiLogin();
-
     // Let's create a team
-    await page.goto("/settings/teams/new");
-
-    await test.step("Managed event option exists for team admin", async () => {
```

Comment:
We already know if the option exists in the "Can create managed event type" test. Removed due redundancy.